src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/XASdkTests.cs

```diff
@@ -67,6 +67,25 @@ public void DotNetBuildBinding ()
 			}
 		}
 
+		[Test]
+		public void DotNetBuildReleaseArm64 ([Values (false, true)] bool forms)
+		{
+			var proj = forms ?
+				new XamarinFormsXASdkProject () :
+				new XASdkProject ();
+			proj.IsRelease = true;
+
+			if (!forms)
+				proj.OtherBuildItems.Add (new AndroidItem.InputJar ("javaclasses.jar") {
+					BinaryContent = () => Convert.FromBase64String (InlineData.JavaClassesJarBase64)
+				});
+
+			proj.SetProperty (KnownProperties.RuntimeIdentifier, "android.21-arm64");
+
+			var dotnet = CreateDotNetBuilder (proj, false);
+			Assert.IsTrue (dotnet.Build (), "`dotnet build` should succeed");
+		}
```

Comment:
I think the `BuildTest.BuildReleaseArm64` test by itself will be sufficient to compare.

We multi-target this project:

https://github.com/xamarin/xamarin-android/blob/b3d2910af1b523fb86d8edeb0ba7770c9446ece9/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Xamarin.Android.Build.Tests.csproj#L4

If you run the `netcoreapp3.1` version, it generates SDK-style projects and builds with `dotnet build`.

We might be missing a way to run this from the command-line easily. I normally use the IDE.

Reply:
I tried to use the `netcoreapp3.1` version on cmd line and it doesn't work with nunit3-console for me. I am getting:
```
NUnit.Engine.NUnitEngineException : The NUnit 3 driver encountered an error while executing reflected code.
  ----> System.InvalidCastException : Unable to cast transparent proxy to type 'System.Web.UI.ICallbackEventHandler'.
--NUnitEngineException
The NUnit 3 driver encountered an error while executing reflected code.
```

So I would like to keep the `DotNetBuildReleaseArm64` test around.

BTW, do you remember why do we need the following part?
```c#
				proj.OtherBuildItems.Add (new AndroidItem.InputJar ("javaclasses.jar") {
					BinaryContent = () => Convert.FromBase64String (InlineData.JavaClassesJarBase64)
				});
```

Reply:
>  it doesn't work with nunit3-console for me

You'll have to use `dotnet test`:

    > dotnet test .\bin\TestDebug\netcoreapp3.1\Xamarin.Android.Build.Tests.dll

> why do we need the following part?

The `@(InputJar)` was in the other test to check that the binding worked:

https://github.com/xamarin/xamarin-android/blob/c81435813751eb9b77f11620513fd4d048b8e209/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/XASdkTests.cs#L145-L151

You can take this part out, I don't think you need to test a binding.

Reply:
Nice! The `dotnet test` works perfectly. I have updated the changes and the description.